### DISCUSSION TITLE
Inserting CSS by using <style>

### DISCUSSION THREAD

**Discussion by taylorhouston** [vanilla-extract-css/vanilla-extract#1063] [Q&A]
Hello, we're building an app at our work in which the contents is being inserted into the DOM to another app. As a result, the css links being relative urls break since its on another site. 

Is there a way to either make the css documents made from vanilla extract absolute or just put the contents into a style tag?

Thanks.

**Comment by askoufis** [comment]
This sounds more like a bundler/framework configuration issue rather than an issue with Vanilla Extract itself, unless I'm misinterpreting your issue. I don't think Vanilla Extract's bundler integrations have any opinions about absolute/relative URLs when they're used in your rendered HTML.
